content/education/manage-coursework-with-github-classroom/teach-with-github-classroom/use-autograding.md

```diff
@@ -15,6 +15,12 @@ redirect_from:
 
 After a student accepts an assignment, on every push to the assignment repository, {% data variables.product.prodname_actions %} runs the commands for your autograding test in a Linux environment containing the student's newest code. {% data variables.product.prodname_classroom %} creates the necessary workflows for {% data variables.product.prodname_actions %}. You don't need experience with {% data variables.product.prodname_actions %} to use autograding.
 
+{% note %}
+
+For more information on workflows and {% data variables.product.prodname_actions %}, see "[AUTOTITLE](/actions/guides/about-continuous-integration).
+
+{% endnote %}
```

Comment:
```suggestion
After a student accepts an assignment, on every push to the assignment repository, {% data variables.product.prodname_actions %} runs the commands for your autograding test in a Linux environment containing the student's newest code. {% data variables.product.prodname_classroom %} creates the necessary workflows for {% data variables.product.prodname_actions %}. You don't need experience with {% data variables.product.prodname_actions %} to use autograding. For more information on workflows and {% data variables.product.prodname_actions %}, see "[AUTOTITLE](/actions/guides/about-continuous-integration)."
```

I would actually say this can also be added in-line with the other content instead of as a note, sorry about that! For the article we looked at together, I think using a note made sense, but I should have taken a look at the other articles before confirming that notes were the way to go with those too 😅